smart-contracts/README.md

```diff
@@ -113,10 +117,44 @@ npx hardhat verify --network polygon 0x7633dd082406861C384309c67576a4260C5775E0
 
 NB: for Polygon, you need an API key from https://polygonscan.com
 
-## Deploy Governor + Timelock
+## Deploy the conctracts
+
+### Deploy all
+
+You can setup an entire test environments with all contracts using
+
+```
+npx hardhat deploy all
+```
+
+Or deploy different part separately
```

Comment:
Really like that granularity!

Reply:
It makes the code slightly more verbose but also more usable